jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyRunMojo.java

```diff
@@ -367,6 +367,7 @@ public void restartWebApp(boolean reconfigure) throws Exception
                 break;
             }
             case HOME:
+            case DISTRO:
```

Comment:
Please produce a warning log message if using DISTRO, encouraging use of HOME instead.